### PR TITLE
Added initial hearing controller validations

### PR DESCRIPTION
**Pull request by BrendanBeachBC**
# Description

This PR includes the following proposed change(s):

- JADE-510

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Unit tests, manual testing.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: pcss-criminal-application/src/main/java/ca/bc/gov/open/pcsscriminalapplication/service/HearingValidator.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package ca.bc.gov.open.pcsscriminalapplication.service;
+
+import ca.bc.gov.open.wsdl.pcss.one.SetHearingRestrictionCriminalRequest;
+
+import java.util.List;
+
+public interface HearingValidator {
+    public List<String> validateSetHearingRestrictionCriminal(SetHearingRestrictionCriminalRequest request);
```
Comment (TayGov) [inline pcss-criminal-application/src/main/java/ca/bc/gov/open/pcsscriminalapplication/service/HearingValidator.java:8]: This doesn't require the public modifier.

Comment (BrendanBeachBC) [reply pcss-criminal-application/src/main/java/ca/bc/gov/open/pcsscriminalapplication/service/HearingValidator.java:8]: I tried removing the public modifier and I'm getting compilation errors. I think it needs to be there.

Comment (TayGov) [reply pcss-criminal-application/src/main/java/ca/bc/gov/open/pcsscriminalapplication/service/HearingValidator.java:8]: Strange I haven't ever done that.

**Review by TayGov — APPROVED** [review]
👍

**Pull request merged**